### PR TITLE
Option for screenshot format

### PR DESCRIPTION
**Pull request by unpause**
You can now change the format in which the screenshot gets saved.
Current options: PNG and JPEG

### CODE REVIEW

File: engine/src/main/java/org/terasology/rendering/opengl/DefaultRenderingProcess.java [hunk 1]
```diff
@@ -1398,7 +1398,8 @@ public void run() {
                 Calendar cal = Calendar.getInstance();
                 SimpleDateFormat sdf = new SimpleDateFormat("yyMMddHHmmss");
 
-                final String fileName = "Terasology-" + sdf.format(cal.getTime()) + "-" + fboSceneFinal.width + "x" + fboSceneFinal.height + ".png";
```
Comment (msteiger) [inline engine/src/main/java/org/terasology/rendering/opengl/DefaultRenderingProcess.java]: I suggest using `new Date()` instead of the Calendar singleton.

**Comment by unpause** [comment]
That Calendar was there before but i can change that later when i have fixed my git...

**Comment by msteiger** [comment]
DateFormat.format() already returns a String, so you don't need to call toString() - but that's ok.

The changes look good to me. There are still merge conflicts which need to be fixed before the code can be merged.

**Comment by unpause** [comment]
i think i fixed them

**Comment by Cervator** [comment]
Tests out good - merged! Thank you :-)

**Pull request merged**